resources/views/partials/forms/edit/location-select.blade.php

```diff
@@ -4,13 +4,12 @@
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
     <div class="col-md-7{{  ((isset($required) && ($required =='true'))) ?  ' required' : '' }}">
         <select class="js-data-ajax" data-endpoint="locations" data-placeholder="{{ trans('general.select_location') }}" name="{{ $fieldname }}" style="width: 100%" id="{{ $fieldname }}_location_select" aria-label="{{ $fieldname }}" {!!  ((isset($item)) && (Helper::checkIfRequired($item, $fieldname))) ? ' data-validation="required" required' : '' !!}{{ (isset($multiple) && ($multiple=='true')) ? " multiple='multiple'" : '' }}>
-            @if ($location_id = old($fieldname, (isset($item)) ? $item->{$fieldname} : ''))
-                <option value="{{ $location_id }}" selected="selected" role="option" aria-selected="true"  role="option">
-                    {{ (\App\Models\Location::find($location_id)) ? \App\Models\Location::find($location_id)->name : '' }}
-                </option>
-            @else
-                <option value=""  role="option">{{ trans('general.select_location') }}</option>
-            @endif
+            {{  ((isset($multiple)) && ($multiple=='true')) ? " multiple='multiple'" : '' }}
```

Comment:
I'm not sure what this check does?

Reply:
So, what we want to do is check that there is a multiple value set, so ignore anything that does NOT ever deal with multiple values. and then also that that value is true.

The original way I wrote it (the in red bits) was how I figured out how to get rid of the bug. It took some trial and error and that's how I came up with the fix, tho it was a bit dupilcative with how it was written.

Addmittedly I am still a bit new with ternary syntax, so I might have oopsed.

Reply:
That line ends up writing `multiple=multiple` onto the page without any effect:
![extra text added to page](https://github.com/snipe/snipe-it/assets/1141514/6e576080-0796-49bb-92ed-cf500e8f48df)

Did you mean to make that an `@isset` blade directive to match the one added below on line 12?